warehouse/query-core/src/main/java/datawave/query/function/KeyToFieldName.java

```diff
@@ -24,43 +24,56 @@ public class KeyToFieldName implements Function<Entry<Key,Value>,Entry<Key,Strin
     
     public KeyToFieldName() {}
     
+    /**
+     * Constructor that sets the includeGroupingContext flag
+     * 
+     * @param includeGroupingContext
+     *            flag that determines if a field name should include the grouping context
+     */
     public KeyToFieldName(boolean includeGroupingContext) {
         this.includeGroupingContext = includeGroupingContext;
     }
     
+    /**
+     * Transforms a key-value entry into a key-string entry
+     * 
+     * @param from
+     *            a key-value entry
+     * @return a key-string entry
+     */
     @Override
     public Entry<Key,String> apply(Entry<Key,Value> from) {
         String fieldName = getFieldName(from.getKey());
-        
         return Maps.immutableEntry(from.getKey(), fieldName);
     }
     
+    /**
+     * Parse the field name from the key's column qualifier
+     * 
+     * @param k
+     *            the key
+     * @return the field name
+     */
     public String getFieldName(Key k) {
         
         int index = -1;
         
-        ByteSequence sequence = k.getColumnQualifierData();
-        
-        byte[] arrayReference = sequence.getBackingArray();
+        ByteSequence bs = k.getColumnQualifierData();
```

Comment:
Aside from the javadoc, the changes here seem kind of pedantic...

Reply:
When key parsing I tend to favor very short names. Range, Key, Row, ColumnFamily, ColumnQualifier, ByteSequence, etc become r, k, row, cf, cq, bs respectively.

I'm certainly willing to make the change if you have a strong preference to the contrary. Otherwise I'll leave it as-is.

Reply:
I won't begrudge your preferences - we all have them.  Clearly the original author had a different set of preferences.  However, if you're going to make more PRs like this, then we should probably get the team leads together and come up with a set of agreed upon coding standards.